components/view/view.js

```diff
@@ -11,6 +11,8 @@
 import { UTILS, DEER } from '//localhost:5500/js/deer-utils.js'
 import '//localhost:5500/components/templates/default.js'
 
+UTILS.worker = await import("../../js/worker.js").then(module => {return module.default}).catch(err => {return {"error":err}})
```

Comment:
This "works" but it is not a `Worker` like `UTILS.worker = new Worker("worker.js'',{type:"module"})` but it may be just fine.